libraries/stdlib/src/kotlin/coroutines/CoroutinesIntrinsicsH.kt

```diff
@@ -44,6 +44,13 @@ public expect inline fun <R, T> (suspend R.() -> T).startCoroutineUninterceptedO
     completion: Continuation<T>
 ): Any?
 
+// Internal version that support arity-2 suspending functions, might be made public in the future if needed
+internal expect inline fun <R, P, T> (suspend R.(P) -> T).startCoroutineUninterceptedOrReturn(
```

Comment:
I'd prefer if we annotated the new internal API with `@SinceKotlin("1.4")` as well, to lessen the probability of accidentally generating access to it in the backend without checking if the API version is at least 1.4.

Reply:
Does the exepect or actual vesion or both need to be annotated this way? Also, it is _inline_ and its actuals are `@InlineOnly`. Does it really matter?

Reply:
It's more of a custom than a requirement. Basically if I'm going to implement some new backend feature in 1.4.x and suddenly realize that using this intrinsic will help -- I'll happily use it. But with `@SinceKotlin("1.4")` on it, I'd think some more and either add a requirement on API version, or avoid using this function.

So since it's for documentation purposes only, I'd annotate both expect and actual. The fact that it's inline-only is not important, since the backend needs to generate a call to some intrinsic in the first place anyway (which can fail later during inlining, e.g. if the function body is not found).